components/proxy-service/internal/metadata/serviceapi/serviceapiservice.go

```diff
@@ -19,7 +19,13 @@ type API struct {
 // Credentials contains OAuth configuration.
```

Comment:
It's not only OAuth anymore :)
Also, we could move API / Credentials / Basic / Oauth to some kind of model package (like in metadata service). What is your opinion?